content/en/blog/_posts/non-root-containers-and-devices.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: blog
+title: 'Non-root Containers And Devices'
+date: 2021-09-YY
+slug: non-root-containers-and-devices
+---
+
```

Comment:
This doesn't read like a blog post, but an academic paper. I'll provide some structural recommendations before I review the content.

Reply:
@shannonxtreme I've now updated the text based on your feedback. Looks better?

Reply:
Flows better for sure (I wouldn't say the text is better because it was already a wonderful contribution and I appreciate you!). I'll do a teeny style/grammar review but they'll be more nits than anything :)